__tests__/app.test.js

```diff
@@ -45,7 +45,7 @@ describe("GET /api/articles/article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body: { article } }) => {
-        expect(article).toEqual(
+        expect(article[0]).toEqual(
```

Comment:
I'd say on this endpoint, it was more appropriate before - a key of article that _**is**_ the article object rather than an array containing it